src/datasources/cache/__tests__/fake.cache.service.spec.ts

```diff
@@ -15,8 +15,9 @@ describe('FakeCacheService', () => {
       faker.string.alphanumeric(),
     );
     const value = faker.string.alphanumeric();
+    const expireTimeSeconds = faker.number.int({ min: 1 });
 
-    await target.set(cacheDir, value, 0);
+    await target.setWithExpiration(cacheDir, value, expireTimeSeconds);
```

Comment:
Nit: let's update the description of this test to reflect that it's with expiration. We should also consider adding a test for `set`. What do you think?

Reply:
Good point! Done in [ec1c982](https://github.com/safe-global/safe-client-gateway/pull/1215/commits/ec1c982243db33ed394ab687d06063e6e0cfbcc3)